static/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "computer-science-flash-cards",
+  "description": "Mini website for testing both general CS knowledge and enforce coding practice and common algorithm/data structure memorization.",
+  "author": "jwasham",
+  "dependencies": {
+    "markdown": "^0.5.0"
```

Comment:
Instead of adding a second package manager, why not install it via python and render the markdown in the back-end?